sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -178,7 +178,7 @@ public virtual Pageable<CustomFormModelInfo> GetModelInfos(CancellationToken can
         [ForwardsClientCalls]
         public virtual AsyncPageable<CustomFormModelInfo> GetModelInfosAsync(CancellationToken cancellationToken = default)
         {
-            return ServiceClient.GetCustomModelsPageableModelInfoAsync(GetModelOptions.Full, cancellationToken);
+            return ServiceClient.GetCustomModelsPageableModelInfoAsync(cancellationToken);
```

Comment:
Are Full/Summary not needed now?

Reply:
No. Now we have different ways to do it. Either by calling `List...` or `Get...` 
See https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json#L457

Reply:
Oh I see how the swagger changed to generate this differently.  I believe where we had Full, we'll want ListCustomModels and where we had Summary, we'll want GetCustomModels based on the new Swagger.

Reply:
Correct